packages/theme/src/components/ui/input/HelperText.vue

```diff
@@ -1,7 +1,21 @@
 <template>
-  <div
-    class="text-sm text-neutral-700"
-  >
+  <div class="text-sm text-right" :class="textColorClass">
+
     <slot />
   </div>
 </template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+
+const props = defineProps({
+  isError: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+const textColorClass = computed(() =>
+  props.isError ? "text-red-600" : "text-neutral-700",
```

Comment:
The red color seems different.

<img width="628" height="300" alt="Image" src="https://github.com/user-attachments/assets/d26dbc66-de37-48bf-9569-f010496e8c96" />